examples/webgl_buffergeometry_selective_draw.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html>
+	<head>
+	<title>three.js - BufferGeometry selectively drawn using attributes and a shader</title>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+	<meta content="initial-scale=1.0, maximum-scale=1.0" name="viewport">
+	<meta name="Generator" content="https://callum.com">
+	<style>
+		body {
+			background-color: #000;
+			font-family: Monospace;
+			margin: 0;
+			color: #000;
+			overflow: hidden;
+		}
+		a {
+			color:#ff0;
+		}
+		#title {
+			position: absolute;
+			width: 100%;
+			top: 0;
+			height: 20px;
+			padding: 0;
+			text-align: center;
+			font-size: 1.1em;
+			background-color: rgba(64,96,64,0.7);
+			color: #fff;
+		}
+        #ui {
+        	position:absolute;
+        	left:0;
+        	top: 20px;
+        	padding: 0;
+        	text-align: center;
+        	width:100%;
+        	height:20px;
+        	color: #fff;
+        	background-color: rgba(64,96,64,0.6);
+		}
+	</style>
+	<script type="text/javascript" src="../build/three.min.js"></script>
+	<script type="text/javascript" src="js/Detector.js"></script>
+	<script type="text/javascript" src="js/libs/stats.min.js"></script>
+
+	<script type="x-shader/x-vertex" id="vertexshader">
+		attribute float visible;
+		varying float vVisible;
+		attribute vec3 vertColor;
+		varying vec3 vColor;
+
+		void main() {
+			gl_Position = projectionMatrix * modelViewMatrix * vec4(position,1.0);
+			vColor = vertColor;
+			vVisible = visible;
+		}
+	</script>
+	<script type="x-shader/x-fragment" id="fragmentshader">
+		varying float vVisible;
+		varying vec3 vColor;
+
+		void main() {
+			if ( vVisible > 0.0 ) {
+				gl_FragColor = vec4( vColor, 1.0 );
+			} else {
+				discard;
+			}
+		}
+	</script>
+	</head>
+	<body onload="app()">
+		<div id="title"></div>
+		<div id="ui"><a href="#" onclick="hide_lines();">CULL SOME LINES</a> - <a href="#" onclick="show_all_lines();">SHOW ALL LINES</a></div>
+		<script>
+		var camera, scene, renderer, stats;
+		var geometry, mesh;
+		var num_lat = 100;
```

Comment:
Will do - I ran it through the DoesItFitMrDoob's coding standards but missed the camelCase requirement - updating now.